fix glitches in attachments list

klyonrad opened the pull request:
This changes the layout of the attachments list (in edit) to an `<ul>` instead of table and removes or changes unnecessary fixed width sizes

Commit:
fix glitches in attachments list

Commit:
add attachment description field and align all elements attachment div correctly

zlorfi reviewed (changes requested):
it is impossible to add more than 2 attachments

Commit:
enable multiple attachments again

The elements have to be built again in JavaScript, the new description parameter has been added there. And .attach class instead of ID is used
